tests/unit_tests/CreateGameController.test.ts

```diff
@@ -0,0 +1,17 @@
+import { CreateGameController } from "../../src/controllers/CreateGameController";
+import { mock, MockProxy } from 'jest-mock-extended';
+import { BoundaryGameDataStruct } from "../../src/use_cases/BoundaryGameDataStruct";
+import { CreateGameUseCase } from "../../src/use_cases/CreateGameUseCase";
+
+describe("Testing game controller", () => {
+    const expectedGameDataStruct = new BoundaryGameDataStruct(1, 'table');
```

Comment:
įkelk šitą į metodą arba pavadink kaip globalią konstantą(ALL_CAPS_WITH_UNDERSCORES)